drivers/timer/altera_avalon_timer_hal.c

```diff
@@ -15,6 +15,8 @@
 
 static u32_t accumulated_cycle_count;
 
+static s32_t _sys_idle_elapsed_ticks = 1;
```

Comment:
The legacy drivers are likely going away and being replaced with much simpler ones written to the new API.   I wanted to avoid needless churn there.  But sure, to the extent we don't want to rewrite timer drivers doing a cleanup pass after all this mess wouldn't be a bad idea.